app/providers/category/[id]/page.tsx

```
@@ -18,116 +14,77 @@ interface CategoryPageProps {
 
 /**
  * Get category name by ID
  */
-async function getCategoryName(categoryId: string): Promise<string> {
-  try {
-    const { categories } = await fetchPublicCategories();
-    const category = categories.find(cat => String(cat.id) === categoryId);
-    return category ? category.name : `Category ${categoryId}`;
-  } catch (error) {
-    console.error("Error fetching category name:", error);
+async function getCategoryName(categoryId: string, categories: any[] = []): Promise<string> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)